Guardfile

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env ruby
```

Comment:
But that's wrong, it's not an executable so this should not get a shebang. for enabling syntax highlighting there are other options.